src/skdh/completeness/__init__.py

```diff
@@ -0,0 +1,25 @@
+from skdh.completeness.complete import *
+from skdh.completeness.helpers import *
+from skdh.completeness.parse import *
+from skdh.completeness.utils import *
+from skdh.completeness.visualizations import *
+
+__all__ = (
```

Comment:
Preferably `__all__` should include only the main module, since users should not need to access the other functions

Reply:
Removed all but completeness_pipe from __all__